features/step_definitions/oauth.rb

```diff
@@ -14,5 +14,6 @@
     opts = {resource: 'oauth', resource_name: name, p: patch_json, type: 'merge' }
     @result = _admin.cli_exec(:patch, **opts)
     raise "Cannot restore OAuth: #{name}" unless @result[:success]
+    sleep 60
```

Comment:
@stuartchuan instead of a hard-coded wait, is there another way to verify `Oauth` is restored properly?

Reply:
Here is the idea come from, https://github.com/openshift/cluster-authentication-operator/blob/master/README.md, the last step. @akostadinov

Reply:
In this case trying to login is even more crap than hardcoding wait :/
I guess we leave it as is for the time being.. unless anybody has some better idea.